packages/@angular/cli/commands/generate.ts

```diff
@@ -34,117 +60,109 @@ export default Command.extend({
       description: 'Run through without making any changes.'
     },
     {
-      name: 'lint-fix',
+      name: 'force',
       type: Boolean,
-      aliases: ['lf'],
-      description: 'Use lint to fix files after generation.'
+      default: false,
+      aliases: ['f'],
+      description: 'Forces overwriting of files.'
     },
     {
-      name: 'verbose',
-      type: Boolean,
-      default: false,
-      aliases: ['v'],
-      description: 'Adds more details to output logging.'
+      name: 'app',
+      type: String,
+      aliases: ['a'],
+      description: 'Specifies app name to use.'
+    },
+    {
+      name: 'collection',
+      type: String,
+      aliases: ['c'],
+      description: 'Schematics collection to use.'
     }
   ],
 
   anonymousOptions: [
-    '<blueprint>'
+    '<schemtatic>'
   ],
 
-  beforeRun: function (rawArgs: string[]) {
-    if (!rawArgs.length) {
-      return;
+  getCollectionName(rawArgs: string[]) {
+    let collectionName = CliConfig.getValue('defaults.schematics.collection');
+    if (rawArgs) {
+      const parsedArgs = this.parseArgs(rawArgs, false);
+      if (parsedArgs.options.collection) {
+        collectionName = parsedArgs.options.collection;
+      }
     }
+    return collectionName;
+  },
+
+  beforeRun: function(rawArgs: string[]) {
+    const collection = getCollection(this.getCollectionName(rawArgs));
 
     const isHelp = ['--help', '-h'].includes(rawArgs[0]);
```

Comment:
Possibly, I want to get it all working first before worrying about tweaking the help output.